@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -737,7 +733,8 @@ export class TextElementsRenderer {
                 }
             }
 
-            if (!this.initializeGlyphs(textElement, textElementStyle)) {
+            const forceNewPassOnLoaded = true;
```

Comment:
Why is this a defined as a constant variable that is always true and not directly calling `!this.initializeGlyhps(textElement, textElementStyle, true)`?

Reply:
It's just a way to self-document code: the constant name already tells you what the boolean does, if you just pass a boolean literal you need to look up the method documentation to see what the boolean means.

Reply:
I see.